core/chains/terra/terratxm/txm.go

```diff
@@ -163,12 +163,28 @@ func (txm *Txm) sendMsgBatch(ctx context.Context) {
 		txm.lggr.Errorw("unable to read unstarted msgs", "err", err)
 		return
 	}
+	var notExpired []terra.Msg
+	var expired []terra.Msg
+	cutoff := time.Now().Add(-txm.cfg.TxMsgTimeout())
+	for _, msg := range unstarted {
+		if msg.CreatedAt.Before(cutoff) {
+			expired = append(expired, msg)
+		} else {
+			notExpired = append(notExpired, msg)
+		}
+	}
+	err = txm.orm.UpdateMsgs(unstarted.GetIDs(), db.Errored, nil)
```

Comment:
Did you consider a specific orm method to do this in SQL? Doesn't seem important to do atomically in the current state, but would prevent accidents in the future.